### PR TITLE
trezor: Use upstream trezor connect.

### PR DESCRIPTION
**Pull request by JoeGruffins**
I'm not sure when it happened, but it looks like they merged my connect work so we can use their `trezor-connect@8.2.4-extended`

@matheusd please have a good look at this if you could, test it out.

I believe it still shouldn't work with stake outputs yet, so needs to be a "clean" trezor.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -318,12 +317,14 @@
     "redux": "^4.0.5",
     "redux-thunk": "^2.3.0",
     "reselect": "^4.0.0",
+    "secp256k1": "^4.0.2",
```
Comment (JoeGruffins) [inline package.json:320]: It wanted this for LN stuff, otherwise I am unable to do "yarn build"

**Comment by JoeGruffins** [comment]
I'm confused about the test failure, it couldn't pull a dependency?

**Comment by JoeGruffins** [comment]
Still failing at an odd place `test/unit/components/views/LNPage/ChannelsTab.spec.js`

**Comment by JoeGruffins** [comment]
Ticket purchasing still seems fine in #2869

**Review by matheusd — APPROVED** [review]
Tested on the emulators for trezor model T and model one.

**Pull request merged**